### PR TITLE
Update 0.tensorflow.Dockerfile

### PR DESCRIPTION
**Pull request by realvz**
*Issue #, if available:*
The image is currently broken. 

*Description of changes:*
There are two typos that break this image. 
Fixed variables:

ARG AWS_OFI_NCCL_VERSION=v1.8.1-aws # it was missing the prefix v'
ARG NCCL_VERSION=v2.18.6-1

and removed the redundant 'v' during NCCL branch switching,

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: 3.test_cases/7.tensorflow-distributed/0.tensorflow.Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM nvcr.io/nvidia/tensorflow:23.10-tf2-py3
 
 ARG EFA_INSTALLER_VERSION=1.30.0
-ARG AWS_OFI_NCCL_VERSION=1.8.1-aws
+ARG AWS_OFI_NCCL_VERSION=v1.8.1-aws
 ARG NCCL_TESTS_VERSION=master
```
Comment (mhuguesaws) [inline 3.test_cases/7.tensorflow-distributed/0.tensorflow.Dockerfile:5]:
```suggestion
ARG NCCL_TESTS_VERSION=v2.13.9
```

File: 3.test_cases/7.tensorflow-distributed/0.tensorflow.Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FROM nvcr.io/nvidia/tensorflow:23.10-tf2-py3
 
 ARG EFA_INSTALLER_VERSION=1.30.0
```
Comment (mhuguesaws) [inline 3.test_cases/7.tensorflow-distributed/0.tensorflow.Dockerfile:3]:
```suggestion
ARG EFA_INSTALLER_VERSION=1.33.0
```

File: 3.test_cases/7.tensorflow-distributed/0.tensorflow.Dockerfile [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-ARG AWS_OFI_NCCL_VERSION=1.8.1-aws
+ARG AWS_OFI_NCCL_VERSION=v1.8.1-aws
 ARG NCCL_TESTS_VERSION=master
 ARG NCCL_VERSION=v2.18.6-1
```
Comment (mhuguesaws) [inline 3.test_cases/7.tensorflow-distributed/0.tensorflow.Dockerfile:6]:
```suggestion
ARG NCCL_VERSION=v2.21.3-1
```

File: 3.test_cases/7.tensorflow-distributed/0.tensorflow.Dockerfile [hunk 4]
```diff
@@ -1,7 +1,7 @@
 FROM nvcr.io/nvidia/tensorflow:23.10-tf2-py3
 
 ARG EFA_INSTALLER_VERSION=1.30.0
-ARG AWS_OFI_NCCL_VERSION=1.8.1-aws
+ARG AWS_OFI_NCCL_VERSION=v1.8.1-aws
```
Comment (mhuguesaws) [inline 3.test_cases/7.tensorflow-distributed/0.tensorflow.Dockerfile:4]:
```suggestion
ARG AWS_OFI_NCCL_VERSION=v1.9.2-aws
```

**Review by mhuguesaws — COMMENTED** [review]
Left comments.

**Comment by perifaws** [comment]
@realvz any progress on the update?

**Comment by realvz** [comment]
Looks good to me. Thanks

**Review by KeitaW — APPROVED** [review]
Thank you for reporting!

**Pull request merged**